validator/src/main/java/com/amazon/aoc/validators/ContainerInsightPrometheusMetricsValidator.java

```diff
@@ -18,14 +19,20 @@
   private final ObjectMapper mapper = new ObjectMapper(new YAMLFactory());
 
   @Override
-  List<Metric> getExpectedMetrics(Context context, String templatePath) throws Exception {
+  List<Metric> getExpectedMetrics(
+      Context context,
+      FileConfig expectedDataTemplate
+  ) throws Exception {
     List<Metric> expectedMetrics = new ArrayList<>();
     List<CloudWatchContext.App> validateApps = getAppsToValidate(context.getCloudWatchContext());
     MustacheHelper mustacheHelper = new MustacheHelper();
     for (CloudWatchContext.App app : validateApps) {
-      String templateInput = mustacheHelper.render(new JsonSchemaFileConfig(
-              FilenameUtils.concat(templatePath,
-                      app.getName() + "_metrics.mustache")), context);
+      FileConfig fileConfig = new LocalPathExpectedTemplate(FilenameUtils.concat(
+          expectedDataTemplate.getPath().getProtocol()
+            + "://"
+            + expectedDataTemplate.getPath().getPath(),
```

Comment:
Can the transformation be simplified as `getPath().toString()`?

Reply:
yes, we can, thanks for pointing out!